### PR TITLE
Fix #78540: Add missing properties during DateInterval dump

### PR DESCRIPTION
**Pull request by carusogabriel**
See https://3v4l.org/Kmt9V for the different outputs across multiple PHP versions.

They were added via php/php-src@0ee71557ffd and php/php-src@55626549d81d.

### CODE REVIEW

**Comment by carusogabriel** [comment]
Also related to [#74054](https://bugs.php.net/bug.php?id=74054).

**Comment by carusogabriel** [comment]
I couldn't figure out what CI is complaining about 😕

**Comment by carusogabriel** [comment]
Looks like my example declaration is wrong:

```sh
ERROR (file:////home/travis/build/php/en/reference/datetime/dateinterval/construct.xml:183:13)
  </example>
-------------^
 Element example content does not follow the DTD, expecting (((title |
 titleabbrev)* , info?) , (itemizedlist | orderedlist | procedure | simplelist |
 variablelist | segmentedlist | glosslist | bibliolist | calloutlist | qandaset |
 caution | important | note | tip | warning | example | figure | table | equation
 | informalexample | informalfigure | informaltable | informalequation | sidebar
 | blockquote | address | epigraph | mediaobject | screenshot | task |
 productionset | constraintdef | msgset | programlisting | screen | literallayout
 | synopsis | programlistingco | screenco | cmdsynopsis | funcsynopsis |
 classsynopsis | methodsynopsis | constructorsynopsis | destructorsynopsis |
 fieldsynopsis | bridgehead | remark | revhistory | indexterm | anchor | para |
 formalpara | simpara | annotation)+ , caption?), got (title programlisting CDATA
 screen )
```

File: reference/datetime/dateinterval/construct.xml [hunk 1]
```diff
@@ -130,20 +130,70 @@
...
     <programlisting role="php">
 <![CDATA[
 <?php
+$interval = new DateInterval('P2Y4DT6H8M');
+
+var_dump($interval);
+?>
+]]>
+    </programlisting>
+    The above example will output for PHP 5.3.27 and PHP 5.4.17 versions and above:
```
Comment (cmb69) [inline reference/datetime/dateinterval/construct.xml]:
```suggestion
   <para>
    The above example will output for PHP 5.3.27 and PHP 5.4.17 versions and above:
   </para>
```

File: reference/datetime/dateinterval/construct.xml [hunk 2]
```diff
@@ -130,20 +130,70 @@
...
+<?php
 
 $interval = new DateInterval('P2Y4DT6H8M');
 var_dump($interval);
 
 ?>
 ]]>
     </programlisting>
-    &example.outputs;
+    The above example will output for PHP 5.3.26 and PHP 5.4.16 versions and bellow:
```
Comment (cmb69) [inline reference/datetime/dateinterval/construct.xml]:
```suggestion
   <para>
    The above example will output for PHP 5.3.26 and PHP 5.4.16 versions and bellow:
   </para>
```

File: reference/datetime/dateinterval/construct.xml [hunk 3]
```diff
@@ -130,20 +130,70 @@
 
  <refsect1 role="examples">
   &reftitle.examples;
-  <para>
-   <example>
+  <example>
     <title><classname>DateInterval</classname> example</title>
```
Comment (cmb69) [inline reference/datetime/dateinterval/construct.xml]: Nit: indentation should be 1 space per level.

**Comment by carusogabriel** [comment]
@cmb69 This patch aims to fix #78540, which the current example is outdated.

#74054 Indeed is something that we need to discuss before documenting any property :)

File: reference/datetime/dateinterval/construct.xml [hunk 4]
```diff
@@ -143,7 +195,9 @@ var_dump($interval);
 ?>
 ]]>
     </programlisting>
-    &example.outputs;
+    <para>
+     The above example will output for PHP 5.3.26 and PHP 5.4.16 versions and bellow:
```
Comment (CJDennis) [inline reference/datetime/dateinterval/construct.xml]: "Below" is spelt with a single "l".

**Pull request closed without merging**